### ISSUE TITLE
formatOptions does not accept lineNumbers

### ISSUE THREAD

**Issue by JSteunou** [lingui/js-lingui#1013]
**Describe the bug**

Documentation says `formatOptions` default value is `{ origins: true, lineNumbers: true }`

Setting those both to false trigger this warning 

```
● Validation Warning:

  Unknown option "formatOptions.lineNumbers" with value false was found.
  This is probably a typing mistake. Fixing it will remove this message.

Documentation: https://lingui.js.org/ref/conf.html
```

**To Reproduce**

using this settings

```
    format: 'po',
    formatOptions: {origins: false, lineNumbers: false},
```

**Expected behavior**

no warning or document in sync with the correct value to use

**Additional context**

- jsLingui version ``lingui --version`` 3.7.2
- Babel version ``npm list @babel/core`` 7.13.10
- Your Babel config (e.g. `.babelrc`) or framework you use (Create React App, Meteor, etc.)

**Comment by semoal** [comment]
This feature wasn't released yet, sorry. 
Released right now with 3.8.0! :)

**Comment by JSteunou** [comment]
Ahah ok. What was the odds hu...

**Comment by nathanforce** [comment]
@semoal This formatting is only happening for `.json` files? Is that intended or shall I open an issue?

**Comment by semoal** [comment]
No, this formatOptions are allowed for `.po` files or lingui format, json files doesn't have extra information about line numbers or origins, just key-value
